src/components/Tooltip/EducationalTooltip.tsx

```diff
@@ -0,0 +1,42 @@
+import React, {memo, useEffect, useRef} from 'react';
+import type {LayoutChangeEvent, LayoutEvent} from 'react-native';
+import BaseTooltip from './BaseTooltip';
+import type TooltipProps from './types';
+
+/**
+ * A component used to wrap an element intended for displaying a tooltip.
+ * This tooltip would show immediately without user's interaction and hide after 5 seconds.
+ */
+function EducationalTooltip({children, ...props}: TooltipProps) {
+    const hideTooltipRef = useRef<() => void>();
+
+    useEffect(() => {
+        if (!hideTooltipRef.current) {
+            return;
+        }
+
+        const intervalID = setInterval(hideTooltipRef.current, 5000);
+        return () => {
+            clearInterval(intervalID);
+        };
+    }, []);
+
+    return (
+        // eslint-disable-next-line react/jsx-props-no-spreading
+        <BaseTooltip {...props}>
+            {({showTooltip, hideTooltip, updateBounds}) => {
+                hideTooltipRef.current = hideTooltip;
+                return React.cloneElement(children as React.ReactElement, {
+                    onLayout: ({nativeEvent}: LayoutEvent | LayoutChangeEvent) => {
+                        updateBounds('target' in nativeEvent && typeof nativeEvent.target !== 'number' ? nativeEvent.target?.getBoundingClientRect() : nativeEvent.layout);
```

Comment:
I don't like this 🙁 This is effectively platform-specific code.

Let's start with this topic: doesn't the `onLayout` abstraction work correctly cross-platform, including React Native Web? Do we actually need the DOM-specific `getBoundingClientRect` call?

Reply:
Oh, I understand! An inconvenient inconsistency 😠

When I understand the technical aspect, I must comment on the code details. It's definitely not a good style to perform dynamic checks when this is platform-specific code. Again, platform-specific files could come to a rescue. There are likely multiple ways to abstract this part, so I'll leave the choice to you.